From: =?utf-8?q?Timo_R=C3=B6hling?= <roehling@debian.org>
Date: Thu, 22 Feb 2024 13:29:06 +0100
Subject: Fudge rust crate version requirements

Forwarded: not-needed
---
 Cargo.toml | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index ed3ea71..0744123 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -29,19 +29,19 @@ include = [
 pyo3 = { version = "0.20.0", features = ["generate-import-lib", "num-bigint"] }
 regex = "1.10.1"
 strum = { version = "0.25.0", features = ["derive"] }
-strum_macros = "0.25.3"
+strum_macros = "0.25.2"
 serde_json = {version = "1.0.107", features = ["arbitrary_precision", "preserve_order"]}
 enum_dispatch = "0.3.8"
 serde = { version = "1.0.189", features = ["derive"] }
-speedate = "0.12.0"
+speedate = "0.13.0"
 smallvec = "1.11.1"
 ahash = "0.8.0"
 url = "2.4.1"
 # idna is already required by url, added here to be explicit
 idna = "0.4.0"
-base64 = "0.21.4"
-num-bigint = "0.4.4"
-python3-dll-a = "0.2.7"
+base64 = "0.21.2"
+num-bigint = "0.4.3"
+python3-dll-a = "0.2.6"
 uuid = "1.4.1"
 
 [lib]
